src/poaps/queries/PaginatedPoaps.ts

```diff
@@ -31,6 +31,46 @@ export const PAGINATED_POAPS_QUERY = /* GraphQL */ `
   }
 `;
 
+export function buildPaginatedPoapsQuery({
+  withMintingStats,
+  withCollectorStats,
+  withDropStats,
+}: {
+  withMintingStats: boolean;
+  withCollectorStats: boolean;
+  withDropStats: boolean;
+}): string {
+  return /* GraphQL */ `
+    query PaginatedPoapsWithStats(
+      $limit: Int!
+      $offset: Int!
+      $orderBy: [poaps_order_by!]
+      $where: poaps_bool_exp
+    ) {
+      poaps(limit: $limit, offset: $offset, order_by: $orderBy, where: $where) {
+        chain
+        collector_address
+        drop_id
+        id
+        minted_on
+        transfer_count
+        drop {
+          image_url
+          city
+          country
+          description
+          start_date
+          end_date
```

Comment:
Timezone is important to display an accurate date range for the drop.
```suggestion
          end_date
          timezone
```